examples/veymont-check/checkLTS/ltstest.pvl

```diff
@@ -1,7 +1,8 @@
-// -*- tab-width:4 ; indent-tabs-mode:nil -*-
 //:: cases LTSMethodCallTest
-//:: tools session
+//:: suite veymont veymont-check
+//:: option --veymont tmp.pvl
 //:: verdict Pass
+//:: tool silicon
```

Comment:
Deze kan weg denk ik? Het is volgens mij niet de bedoeling dat deze met silicon wordt gecheckt :)